TerriaJS/terriajs
# Wc play story btn

### What this PR does

Fixes #3827

Adds play story button in mobile view when there is an active story
  
### Test me
 
Go to : http://ci.terria.io/wc-play-story-btn/
1. Create a story
2. Copy the generated story link onto your clipboard
3. Open the link on a mobile or in a browser window with a narrow viewport
4. Observe the play story prompt come up
5. Observe that the new story button in the bottom left in the default grey
6. Dismiss the prompt
7. Observe that the play story button is in an active blue
8. Click on the play story button
9. Observe that the story is now playing from the first scene



### Checklist
